traitsui/qt4/list_editor.py

```diff
@@ -96,20 +98,19 @@ def init(self, parent):
             trait_handler = self.object.base_trait(self.name).handler
         self._trait_handler = trait_handler
 
-        # Create a scrolled window to hold all of the list item controls:
-        self.control = QtGui.QScrollArea()
-        self.control.setFrameShape(QtGui.QFrame.NoFrame)
-        self.control.setWidgetResizable(True)
+        if self.scrollable:
+            # Create a scrolled window to hold all of the list item controls:
+            self.control = QtGui.QScrollArea()
+            self.control.setFrameShape(QtGui.QFrame.NoFrame)
+            self.control.setWidgetResizable(True)
+        else:
+            self.control = QtGui.QWidget()
 
         # Create a mapper to identify which icon button requested a contextmenu
         self.mapper = QtCore.QSignalMapper(self.control)
 
         # Create a widget with a grid layout as the container.
-        self._list_pane = QtGui.QWidget()
-        self._list_pane.setSizePolicy(
-            QtGui.QSizePolicy.Expanding, QtGui.QSizePolicy.Expanding
-        )
-        layout = QtGui.QGridLayout(self._list_pane)
+        self._layout = layout = QtGui.QGridLayout(self._list_pane)
```

Comment:
`_layout` isn't declared as a trait on the class (but I don't think you actually need it - see previous comment).